src/Compilers/CSharp/Portable/Binder/ForEachLoopBinder.cs

```diff
@@ -753,7 +753,7 @@ private EnumeratorResult GetEnumeratorInfo(ref ForEachEnumeratorInfo.Builder bui
                 return EnumeratorResult.FailedAndReported;
             }
 
-            if (AllInterfacesContainsIEnumerable(ref builder, collectionExprType, isAsync, diagnostics, out foundMultipleGenericIEnumerableInterfaces))
+            if (AllInterfacesContainsIEnumerable(ref builder, collectionExprType, isAsync, diagnostics, out bool foundMultipleGenericIEnumerableInterfaces))
```

Comment:
The out parameter declaration was inlined, as somehow the declaration had moved over 30 lines up the file, and was confusing.